source/code-snippets/usage-examples/insertMany.ts

```diff
@@ -0,0 +1,36 @@
+import { MongoClient } from "mongodb";
+
+// Replace the uri string with your MongoDB deployment's connection string.
+const uri =
+  "mongodb+srv://<user>:<password>@<cluster-url>?writeConcern=majority";
```

Comment:
From the discussion yesterday, I think the parameter name should be "w" instead of "writeConcern" (see https://github.com/mongodb/specifications/blob/master/source/uri-options/uri-options.rst). I think the latter is an artifact of the 3.6 Node driver.

Reply:
This was discussed in slack and captured in [this JIRA ticket](https://jira.mongodb.org/browse/DOCSP-18084)